docs/community.rst

```diff
@@ -0,0 +1,43 @@
+=========
+Community
+=========
+
+.. todo:: 
+
+    This is just a placeholder page to explain the use and development of rasterio.
+    
+    
+Who is using rasterio?
+----------------------
+
+* `Mapbox`_ uses rasterio to process satellite imagery for it's global satellite base maps.
+* Mary Marek-Spartz's masters thesis compares the map algebra tools of the proprietary ESRI ArcPy library and the open source Rasterio library.  https://github.com/alfalimajuliett/masters-thesis
+
```

Comment:
@perrygeo might be nice to include a brief statement on how to get your project or use case listed here.

Please add:
* The Conservation Biology Institute (http://consbio.org/) uses rasterio to provide geoprocessing of NetCDF files (https://github.com/consbio/clover).